test/boulder-config.json

```diff
@@ -116,7 +116,8 @@
   },
 
   "pa": {
-    "dbConnect": "mysql+tcp://policy@localhost:3306/boulder_policy_integration"
+    "dbConnect": "mysql+tcp://policy@localhost:3306/boulder_policy_integration",
+    "challengeTypes": ["simpleHttp", "http-01", "tls-sni-01"]
```

Comment:
I'm pretty sure that the `boulder-ca` could get away with setting this to the empty array (or omitting it), though I'd want to test that.  After all, it only gets used in `ChallengesFor()`, which the CA doesn't use.